modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -376,7 +389,8 @@ private String _formatParameters(
 			0, to.indexOf(CharPool.OPEN_PARENTHESIS) + 1);
 
 		if (!newMethodCall.contains(StringPool.PERIOD) &&
-			!Character.isUpperCase(newMethodCall.charAt(0))) {
+			!Character.isUpperCase(newMethodCall.charAt(0)) &&
+			!newMethodCall.contains(StringPool.SPACE)) {
```

Comment:
@nicolas-sss, why do you need the blank space?

Reply:
For the scenarios where we don't want to get the variableName, that is what is done inside that if, when it has a period, the first letter is Upper case or it has a space character.